source/core/icosahedron.ts

```diff
@@ -0,0 +1,212 @@
+
+import { vec2, vec3 } from 'gl-matrix';
+import { v3 } from '../gl-matrix-extensions';
+
+type ivec3 = [number, number, number];
+
+/**
+ * Helper class to generate a sphere geometry based on a refinable icosahedron
+ */
+export class Icosahedron {
```

Comment:
extend ```Geometry```

Reply:
This is actually just a helper class to initialize `Sphere` and does not represent a drawable geometry itself